clients/include/testing_gemm_strided_batched.hpp

```diff
@@ -238,7 +238,7 @@ void testing_gemm_strided_batched(const Arguments& arg)
 
     if(arg.timing)
     {
-        int number_cold_calls = 2;
+        int number_cold_calls = arg.iters;
```

Comment:
should be `cold_iters`

Reply:
Shouldn't that be `arg.cold_iters`, or is it a special case?

Reply:
fixed in last commit